src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
+    public Boolean isMutating() {
+        return true;
+    }
+
```

Comment:
Do remember to give header comments for methods, it helps to give people an idea of what the method is trying to do, especially overridden methods.

Similar for other isMutating() methods in other classes